### PR TITLE
Now it's possible to manage your address book (user addresses)

### PR DESCRIPTION
**Pull request by maerteijn**

### CODE REVIEW

**Comment by specialunderwear** [comment]
I think it still needs to be proven that you can not change the user an address belongs to, in that way messing with anothers users adresses.

**Comment by maerteijn** [comment]
@specialunderwear Like this https://github.com/django-oscar/django-oscar-api/pull/97/commits/bc901bee72b66d951f6258849efd8f184a92d807

File: oscarapi/tests/testuseraddress.py [hunk 1]
```diff
@@ -103,3 +106,55 @@ def test_useraddress_update_and_delete(self):
...
+    def test_address_security(self):
+        "Make sure we can't access adresses of other users"
+        # first login as the first user
+        self.login('nobody', 'nobody')
+
+        nobodys_address = deepcopy(ADDRESS)
+        nobodys_address['last_name'] = 'Nobody'
+        url = reverse('useraddress-list')
+        self.response = self.post(url, **nobodys_address)
+
```
Comment (specialunderwear) [inline oscarapi/tests/testuseraddress.py]: how about:
```
nobodys_address['user'] = 3 # id of other user which I guessed.
self.put(url, **nobodys_address)  # now I get his deliveries HAHAHAHAHAHAAHAHA pwnd
```

Comment (maerteijn) [inline oscarapi/tests/testuseraddress.py]: https://github.com/django-oscar/django-oscar-api/pull/97/commits/0acc4b689b638bae1e8267bfedceb4afcc3444ee

**Pull request merged**